src/main/java/com/infamous/combat_next/config/MeleeCombatConfigs.java

```diff
@@ -53,6 +63,18 @@ static void createServerConfigs(ForgeConfigSpec.Builder builder) {
                             For vanilla, this value is false.
                             """)
                     .define("attack_during_cooldown_prevented", true);
+            attackGracePeriod = b
+                    .comment("""
+                            Toggles the "grace period" where if you attack before 100% when you shouldn't be able to, but are between 80% and 100%, the attack is delayed until one tick after 100%.
```

Comment:
Change the 80% value here to specify it is the value in the following attackGracePeriodTime config

Reply:
makes sense

Reply:
wrote it before adding the time config